kong/plugins/ldap-auth/handler.lua

```diff
@@ -1,18 +1,23 @@
 local access = require "kong.plugins.ldap-auth.access"
 local BasePlugin = require "kong.plugins.base_plugin"
 
+
 local LdapAuthHandler = BasePlugin:extend()
 
+
 function LdapAuthHandler:new()
   LdapAuthHandler.super.new(self, "ldap-auth")
 end
 
+
 function LdapAuthHandler:access(conf)
   LdapAuthHandler.super.access(self)
   access.execute(conf)
 end
 
+
 LdapAuthHandler.PRIORITY = 1002
-LdapAuthHandler.VERSION = "0.1.0"
+LdapAuthHandler.VERSION = "0.1.1"
```

Comment:
All of these massive updates warrant increasing the second digit (e.g. `0.2.0`) in all of these plugins, instead of the patch digit.